environment.yml

```diff
@@ -16,15 +18,15 @@ dependencies:
 - psutil
 - numpy
 - scipy
-- xarray==0.17
-- xclim
+- xarray >=0.18
+- xclim >=0.28.1
 - rioxarray
 - matplotlib
 - dask
-- pandas
+- pandas <1.3
```

Comment:
Is this still necessary ?

Reply:
No, the bug has been resolved. Will modify.